File: faktorial.c
#include <stdio.h>

int fak(int a){
    long hasil = 1;
    for(int i=1; i<=a; i++){
        hasil *= i;
    }
    return hasil;
}

int main(){
    int a;
    scanf("%d", &a);
    printf("%d", fak(a));
    return 0;
}

File: faktorial.dart
int fak(int a) {
  var hasil = 1;
  for (var i = 1; i <= a; i++) {
    hasil *= i;
  }
  return hasil;
}

void main() {
  //kasus faktorial angka 6
  print("Faktorial 6 : ${fak(6)}");
}
